streams/src/main/java/org/apache/kafka/streams/processor/internals/StandbyTask.java

```diff
@@ -144,8 +162,18 @@ public void close(final boolean clean) {
         return stateMgr.updateStandbyStates(partition, records);
     }
 
-    Map<TopicPartition, Long> checkpointedOffsets() {
+    @Override
+    public int addRecords(final TopicPartition partition, final Iterable<ConsumerRecord<byte[], byte[]>> records) {
```

Comment:
I think `StandbyTask#addRecords` will never be called, so it is OK to be more strict and throw unsupported exception here?